code-challenges.test.js

```diff
@@ -14,43 +14,78 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibonacciLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+describe("fibonacciSeq", () => {
+  it("takes in a number (greater than 2) and returns an array containing the Fibonacci sequence", () => {
+    const fibonacciLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    const fibonacciLength2 = 10
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(fibonacciSeq(fibonacciLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibonacciSeq(fibonacciLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
 
-const fibonacciLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// Output: ReferenceError: fibonacciSeq is not defined
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+const fibonacciSeq = (number) => {
+  // Initialize the first two Fibonacci numbers
+  FibPair = [1, 1]  
+  const FibPair = [1,1]
```

Comment:
Remember to use appropriate naming conventions. We would want this to be called `fibPair`